framework/src/play/src/main/scala/play/api/Configuration.scala

```diff
@@ -37,8 +41,9 @@ object Configuration {
         devSettings.get("config.file").orElse(Option(System.getProperty("config.file")))
           .map(f => new File(f)).getOrElse(new File(appPath, "conf/application.conf"))
       }
+      val playOverridesConfig: Config = getPlayOverridesConfig(Thread.currentThread().getContextClassLoader())
```

Comment:
Actually you may not be. GAA! I may be confounding issues - I was looking at an issue that Johan raised related to activator but now that I think about it that may have been against 15v09.  Let me confirm, this may not be an issue.